src/Symfony/Component/AssetMapper/ImportMap/ImportMapGenerator.php

```diff
@@ -223,7 +229,12 @@ private function findEagerImports(MappedAsset $asset): array
             $dependencies[] = $javaScriptImport->importName;
 
             // Follow its imports!
-            $dependencies = array_merge($dependencies, $this->findEagerImports($javaScriptImport->asset));
+            $nextAsset = $this->assetMapper->getAsset($javaScriptImport->assetLogicalPath);
+            if (!$nextAsset) {
```

Comment:
my usual (but consistent!) random comments :P
```suggestion
            if (!$nextAsset = $this->assetMapper->getAsset($javaScriptImport->assetLogicalPath)) {
```